ee/vellum_ee/workflows/display/nodes/vellum/templating_node.py

```diff
@@ -36,9 +40,10 @@ def serialize(
                 input_name=variable_name,
                 value=variable_value,
                 display_context=display_context,
-                input_id=self.input_ids_by_name.get("template"),
+                input_id=self.input_ids_by_name.get(variable_name) or self.node_input_ids_by_name.get(variable_name),
```

Comment:
another bug I noticed. is the convention meant to be `node_input_ids_by_name` or `template_input_id` + `input_ids_by_name` ?

Reply:
I think it's supposed to be the former